staging/src/k8s.io/kube-aggregator/pkg/registry/apiservice/etcd/etcd.go

```diff
@@ -53,6 +57,60 @@ func NewREST(scheme *runtime.Scheme, optsGetter generic.RESTOptionsGetter) *REST
 	return &REST{store}
 }
 
+var swaggerMetadataDescriptions = metav1.ObjectMeta{}.SwaggerDoc()
+
+func (c *REST) ConvertToTable(ctx context.Context, obj runtime.Object, tableOptions runtime.Object) (*metav1beta1.Table, error) {
+	table := &metav1beta1.Table{
+		ColumnDefinitions: []metav1beta1.TableColumnDefinition{
+			{Name: "Name", Type: "string", Format: "name", Description: swaggerMetadataDescriptions["name"]},
+			{Name: "Service", Type: "string", Description: "The reference to the service that hosts this API endpoint."},
+			{Name: "Available", Type: "string", Description: "Whether this service is available."},
+			{Name: "Age", Type: "string", Description: swaggerMetadataDescriptions["creationTimestamp"]},
```

Comment:
didn't we have some special format or type for the age column to be printed properly?

Reply:
I'm not sure we actually implemented it.  I think it would be type=string, format=age or format=duration.  I can do a follow up PR and set format duration on all the things (every age column)

Reply:
An age string is not a date though.

Reply:
on the wire it's a date, in the kubectl output it's an age. Yes, strange.

Reply:
On the wire it's `row.Cells, err = rowFn(obj, m, m.GetName(), ConvertToHumanReadableDateType(m.GetCreationTimestamp()))` on all types

Reply:
@soltysh you wanted to look into the current code? Anything to add? I don't want to block this due to my comment. Just had the impression we use "date" elsewhere.

Reply:
I did, I'm fine with the current code. It is consistent with the others.

Reply:
:+1: